### PR TITLE
Updated vb_script from last release

### PR DESCRIPTION
**Pull request by Jaybsoni**
**Context:** 
Updated the vb script to the latest version since we did not release this plugin during the last PL release

### CODE REVIEW

File: .github/workflows/vb_script.py [hunk 1]
```diff
@@ -13,26 +13,23 @@
...
+    """
+    clean_str = my_str.replace('"', '')  # remove "
+    clean_str = clean_str.replace("'", "")  # remove '
+    return clean_str
 
 
 def bump_version(version_line, pre_release):
     """ A helper function which takes the current version string and
     replaces it with the bumped version depending on the pre/post
     release flag.
```
Comment (Jaybsoni) [inline .github/workflows/vb_script.py:32]: Example is good
```suggestion
    Example:
         Assume that we have PennyLane version 0.18.0 as the latest release on PyPi.
         Pre-release:
         We gather the version number for the latest of PennyLane release (i.e., v0.18.0) and increment the middle value
         to obtain the version of the plugin that we'll release: v0.19.0.
         Post-release:
         We have just released the v0.19.0 version of the plugin and have v0.19.0 set as the version number.
         We are entering a new development phase of the plugin attached to the next release, so we increment the
         version number (0.20.0) and add the "-dev" suffix (0.20.0-dev).

```

File: .github/workflows/vb_script.py [hunk 2]
```diff
@@ -22,16 +31,16 @@ def bump_version(version_line, pre_release):
...
 
     Example:
-        Assume that we have PennyLane version 0.18.0 as the latest release on PyPi.
+         Assume that we have PennyLane version 0.18.0 as the latest release on PyPi.
```
Comment (Jaybsoni) [inline .github/workflows/vb_script.py:34]:
```suggestion
        Assume that we have PennyLane version 0.18.0 as the latest release on PyPi.
```

File: .github/workflows/vb_script.py [hunk 3]
```diff
@@ -22,16 +31,16 @@ def bump_version(version_line, pre_release):
...
+         to obtain the version of the plugin that we'll release: v0.19.0.
 
-        Post-release:
-        We have just released the v0.19.0 version of the plugin and have v0.19.0 set as the version number.
-        We are entering a new development phase of the plugin attached to the next release, so we increment the
-        version number (0.20.0) and add the "-dev" suffix (0.20.0-dev).
+         Post-release:
+         We have just released the v0.19.0 version of the plugin and have v0.19.0 set as the version number.
+         We are entering a new development phase of the plugin attached to the next release, so we increment the
+         version number (0.20.0) and add the "-dev" suffix (0.20.0-dev).
```
Comment (Jaybsoni) [inline .github/workflows/vb_script.py:43]:
```suggestion
        Post-release:
        We have just released the v0.19.0 version of the plugin and have v0.19.0 set as the version number.
        We are entering a new development phase of the plugin attached to the next release, so we increment the
        version number (0.20.0) and add the "-dev" suffix (0.20.0-dev).
```

**Review by antalszava — APPROVED** [review]
Looks good :+1: Thanks!

**Pull request merged**